templates/repo/projects/view.tmpl

```diff
@@ -150,13 +150,21 @@
 								</span>
 								<a class="project-board-title" href="{{$.RepoLink}}/issues/{{.Index}}">#{{.Index}} {{.Title}}</a>
 							</div>
+							{{- if .MilestoneID }}
 							<div class="meta">
-								{{ if .MilestoneID }}
 								<a class="milestone" href="{{$.RepoLink}}/milestone/{{ .MilestoneID}}">
 									{{svg "octicon-milestone"}} {{ .Milestone.Name }}
 								</a>
-								{{ end }}
 							</div>
+							{{- end }}
+							{{- range index $.LinkedPRs .ID }}
+							<div class="meta">
+								<a href="{{$.RepoLink}}/pulls/{{ .ID }}">
+									<span class="{{if .PullRequest.HasMerged}}purple{{else if .IsClosed}}red{{else}}green{{end}}">{{svg "octicon-git-merge"}}</span>
```

Comment:
I am slightly concerned about deferencing `.PullRequest` without a nil check here. While this should not be a problem, since the issue list is only built from references which have `comment.RefIsPull`, it is not a strict check. What do you think?

Reply:
I have added a check in https://github.com/roobre/gitea/commit/e014aa4effa4a772d87f3e6317db3125aaad1a31, now only Issues which are PRs are fetched from the DB. While this is still not a strict nil check, it should be stronger than before.